gym/envs/toy_text/frozen_lake.py

```diff
@@ -61,11 +61,11 @@ def generate_random_map(size: int = 8, p: float = 0.8) -> List[str]:
         A random valid map
     """
     valid = False
-    board = []  # initialize to make pyright happy
+    board: List[List[str]] = []  # initialize to make pyright happy
```

Comment:
`np.ndarray`? Though from experience typing with numpy is a pain

Reply:
I can try it with a numpy array. It was initialized to a list because the validation function, `is_valid`, expects a `List[List[str]]`

Reply:
It works fine with a numpy array 🙂 
I had to change the `is_valid` function to accept `np.ndarray` instead of  `List[List[str]]`

Reply:
Sorry, I thought they overwrote the board with a numpy. If they actually use a `list[list[str]]` keep it as that

Reply:
Okay